doc/changelog/06-Ltac2-language/18095-br-ltac2-extensions.rst

```diff
@@ -0,0 +1,9 @@
+- **Added:**
```

Comment:
You can just add another item ie
```suggestion
- **Changed:** type of fold_left
- **Added:**
```
or you can manually create a sibling file `18095-whatever.rst` (the name of the file does not really matter, the number impacts the ordering in the combined changelog but that's not very important and the rest is just to be able to tell what the file is about without opening it and without opening the associated PR)